release-notes/versioned/pulsar-2.9.5.md

```diff
@@ -0,0 +1,123 @@
+---
+id: pulsar-2.9.5
+title: Apache Pulsar2.9.5
+sidebar_label: Apache Pulsar 2.9.5
+---
+
+#### 2023-4-23
+
+### Broker
+
+* [improve][broker] Omit making a copy of CommandAck when there are no broker interceptors [#18997](https://github.com/apache/pulsar/pull/18997)
+* [fix][broker] Copy proto command fields into final variables in ServerCnx [#18987](https://github.com/apache/pulsar/pull/18987)
+* [improve][broker] Add logs for why namespace bundle been split (#19003)
+* [fix][broker] Fix deadlock in PendingAckHandleImpl (#18989)
+* [fix][broker]Update interceptor handler exception (#18940)
+* [improve][broker] Make Consumer#equals more effective (#18662)
+* [fix][broker]add test case for deleting namespace fail when has partitioned system topic (#17338)
+* [fix][broker] Fix open cursor with null-initialPosition result with earliest position (#18416)
+* [improve][broker] Add ref count for sticky hash to optimize the performance of Key_Shared subscription (#19167)
+* [fix][broker] Support deleting partitioned topics with the keyword -partition- (#19230)
+* [improve][broker] Follow up #19230 to tighten the validation scope (#19234)
+* [branch-2.9] Fix compile error caused by cherry-pick #19167. (#19241)
+* [improve][websocket][branch-2.9] Add ping support (#19245)
+* [fix][ml] Topics stats shows msgBacklog but there reality no backlog (#19275)
+* [fix][websocket][branch-2.9] Fix webSocketPingDurationSeconds config (#19293)
+* [fix] [ml] Fix the incorrect total size if use ML intercepto (#19404)
+* [fix][broker] Expect msgs after server initiated CloseProducer (#19446)
+* [cleanup][broker] Validate originalPrincipal earlier in ServerCnx (#19270)
+* [fix][Branch-2.9] Fix bookie JVM param BOOKIE_MEM, BOOKIE_GC miss problem. #19441
+* [improve][broker] ServerCnx: go to Failed state when auth fails (#19312)
+* [fix][broker][branch-2.9] Fix geo-replication admin (#19614)
+* [fix] [ml] messagesConsumedCounter of NonDurableCursor was initialized incorrectly (#19355)
+* [fix][ml] Fix potential NPE cause future never complete. (#19415)
+* [fix] [ml] topic load fail by ledger lost (#19444)
+* [fix] [broker] getLastMessageId returns a wrong batch index of last message if enabled read compacted (#18877)
+* [fix][broker] Remove timestamp from broker metrics (#17419)
+* [fix][broker]fix multi invocation for ledger createComplete (#18975)
```

Comment:
```suggestion
* [fix][broker]Fix multi invocation for ledger createComplete (#18975)
```
Please keep consistent on the first letter of a sentence and check all occurrences

Reply:
This comment still needs to be resolved. Please keep consistent on the first letter of a sentence and check all occurrences, then ask for a review. Thanks.

Reply:
Ok

Reply:
@Anonymitaet I check all occurrences and fixed them, please take a look again.